### PR TITLE
BG-79341-sei-sdk-addition

### PR DESCRIPTION
**Pull request by DinshawKothari**
Ticket: BG-79341

### CODE REVIEW

File: modules/sdk-coin-sei/src/tsei.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Testnet Sei
+ *
+ * @format
```
Comment (dpkjnr) [inline modules/sdk-coin-sei/src/tsei.ts:4]: any use of this? 🤔

File: modules/sdk-coin-sei/test/integration/index.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import 'should';
+
+import { BitGoAPI } from '@bitgo/sdk-api';
+import { TestBitGo, TestBitGoAPI } from '@bitgo/sdk-test';
+
+import { Sei, Tsei } from '../../src/index';
+
+describe('SEI', function () {
```
Comment (dpkjnr) [inline modules/sdk-coin-sei/test/integration/index.ts:8]: it doesn't have any testcase?

File: modules/sdk-coin-sei/src/tsei.ts (hunk 1 above)
Comment (DinshawKothari) [reply modules/sdk-coin-sei/src/tsei.ts:4]: Not sure what is the use. It is added in all Testnet coin files. Hence added it.

File: modules/sdk-coin-sei/test/integration/index.ts (hunk 2 above)
Comment (DinshawKothari) [reply modules/sdk-coin-sei/test/integration/index.ts:8]: Yes, But other sdks have this kind of placeholder for integration tests. May be if we want to add integration tests in future.

File: modules/sdk-coin-sei/test/unit/transactionBuilder/transactionBuilder.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    bitgo.safeRegister('tsei', Tsei.createInstance);
+    bitgo.initializeTestVars();
+    basecoin = bitgo.coin('tsei');
+    factory = basecoin.getBuilder();
+  });
+
+  const testTxData = testData.TEST_SEND_TX;
+  let data;
+
+  beforeEach(() => {
```
Comment (dpkjnr) [inline modules/sdk-coin-sei/test/unit/transactionBuilder/transactionBuilder.ts:25]: why not `before`? we are not updating anything

**Review by dpkjnr — APPROVED** [review]

**Pull request merged**